kafka_consumer/tests/test_kafka_consumer.py

```diff
@@ -34,22 +34,22 @@ def test_gssapi(kafka_instance, dd_run_check):
 @pytest.mark.unit
 def test_tls_config_ok(kafka_instance_tls):
     with mock.patch('datadog_checks.base.utils.tls.ssl') as ssl:
-        with mock.patch('kafka.KafkaClient') as kafka_client:
+        with mock.patch('kafka.KafkaAdminClient') as kafka_admin_client:
 
             # mock Kafka Client
-            kafka_client.return_value = mock.MagicMock()
+            kafka_admin_client.return_value = mock.MagicMock()
 
             # mock TLS context
             tls_context = mock.MagicMock()
             ssl.SSLContext.return_value = tls_context
 
             kafka_consumer_check = KafkaCheck('kafka_consumer', {}, [kafka_instance_tls])
-            kafka_consumer_check._create_kafka_client(clazz=kafka_client)
+            kafka_consumer_check.client._create_kafka_client(clazz=kafka_admin_client)
```

Comment:
Does this test still fail when `_create_kafka_client()` returns `KafkaAdminClient`?
For example, if we change [this](https://github.com/DataDog/integrations-core/blob/ad3873fe5875a077ecfb9f913cff1a0cef13a5e2/kafka_consumer/datadog_checks/kafka_consumer/client/kafka_python_client.py#L257) to return `KafkaAdminClient` instead of `clazz`

Reply:
The test will no longer work if we remove `clazz` as an argument, since this will not let us control the type of object we will want to create (we create a mocked `KafkaAdminClient` and have the check return that rather than a real `KafkaAdminClient` object).